vendor/github.com/modern-go/reflect2/unsafe_link.go

```diff
@@ -42,9 +36,21 @@ func ifaceE2I(rtype unsafe.Pointer, src interface{}, dst unsafe.Pointer)
 // If you modify hiter, also change cmd/internal/gc/reflect.go to indicate
 // the layout of this structure.
 type hiter struct {
```

Comment:
are all of these updated fields correct/compatible with versions of go prior to 1.18?

Reply:
this is the only remaining question I have

Reply:
verified the runtime.hiter struct did not change members between go1.17 and master